### PR TITLE
more pipeline training

### PR DESCRIPTION
**Pull request by czchou2**
more spacy pipeline training using scraped and processed tweets on voter fraud, 2020 election, and abortion

61% accurate

E: epochs, #: training steps
LOSS TEXTCAT: penalty for incorrect prediction
CATS SCORE: text cat score
SCORE: weighted score (only training one component, same as cats score)

Pasted Graphic

Results
TOK: tokenized all words
TEXTCAT: overall score of 61%
SPEED: words/second processed

Textcat F
precision: accuracy of predicted positives out of true and false positives
recall: accuracy of true positives out of true positives and false negatives
f score: function of precision and recall 

all metrics were extremely low for negative labels

ROC AUC: fairly high for all labels (high probability of being classified correctly)

TEXTCAT (macro F) 61 10

no lint issues on training2.py, test_clean passes
test_scrape is failing. were new changes made to the scrape function @17chua @zhangathan-jon  ?

Pasted Graphic 2

### CODE REVIEW

**Comment by jl9306** [comment]
Looks like an output file is now required, everything aside from that looks good though.
